numpy/core/src/umath/ufunc_object.c

```diff
@@ -5396,6 +5396,8 @@ ufunc_outer(PyUFuncObject *ufunc, PyObject *args, PyObject *kwds)
     PyArrayObject *ap1 = NULL, *ap2 = NULL, *ap_new = NULL;
     PyObject *new_args, *tmp;
     PyObject *shape1, *shape2, *newshape;
+    PyObject *numpy = PyImport_ImportModule("numpy");
```

Comment:
I can't imagine this will help performance any for ufunc method--though these kinds of imports are littered throughout NumPy source